### DISCUSSION TITLE
Error: Refusing to broadcast a reading as no sensor is active

### DISCUSSION THREAD

**Discussion by libxmike** [NightscoutFoundation/xDrip#3192] [Bug reports]
After fiddling with few phones and after fresh xDrip installation and importing settings from working phone and setting everything correct, the BG was not broadcast with the error in title. The phone was set as xDrip follower.
The only solution I've found was exporting the xDrip DB from working phone and importing on new one, so the Sensor start is not exported/imported in the settings, the Sensor start was not set on the System Status page on xDrip.
After importing the xDrip DB, everything works and the Sensor Start is set and AAPS is receiving BD from xDrip.

![Screen Shot 2023-11-17 at 15 19 37 PM](https://github.com/NightscoutFoundation/xDrip/assets/16604543/5ed483a8-c571-4790-8eaa-f4c44d8532bb)
![Screen Shot 2023-11-17 at 15 26 46 PM](https://github.com/NightscoutFoundation/xDrip/assets/16604543/98a0fe1c-abf4-4418-9b6d-f5ed17272693)

**Comment by jamorham** [comment]
Can you describe your data source and setup in more detail?

**Reply by libxmike** [reply]
BG data comes from my master (OnePlus 9, latest nightly xDrip + AAPS 3.2.0.2) it is set as "Master for followers" with Dexcom G6 
1st follower (Moto Z3) latest nightly xDrip, it is my test phone for AAPS, no issue, but not sure if was on when I changed the sensor.

the problem -> 2nd follower Sony NW-A105, new xDrip installation with imported settings from 1st follower (transferred via USB stick as the NW-A105 does not have camera to scan QR), there was no "Sensor Start" date, only after importing the DB from 1st follower.
Same issue on my 3rd test/follower.

All devices on same network and same "Handset Group key"

**Reply by libxmike** [reply]
what I've also noticed in the error log, "Received a GB reading that appears to be in the future", but it is the same to the second. Can be that xDrip is working with Epoch time or with milliseconds? 

This issue is not related to the other one, get it even after DB import and when everything else is working, BUT the error occurred sporadically, not on every reading.

![Screen Shot 2023-11-20 at 07 14 55 AM](https://github.com/NightscoutFoundation/xDrip/assets/16604543/c3d1ae80-8a78-4211-a729-fa877ca6d211)
![Screen Shot 2023-11-20 at 07 14 24 AM](https://github.com/NightscoutFoundation/xDrip/assets/16604543/5563181e-b619-492b-bbfd-8beed8350adb)

**Comment by Navid200** [comment]
@libxmike 

Do I understand correctly that the master is an xDrip collector from a G6 transmitter?
If yes, can you tell us if the master is working successfully or not?

If no, can you explain what is collecting from G6?

If the problem is setting up an xDrip sync follower that does not have a camera, please follow the instructions on the following link using the manual method:
https://navid200.github.io/xDrip/docs/xDripSync.html

**Reply by libxmike** [reply]
The master phone is working w/o issue, as it always was.

As stated, the follower was not working as expected (just the broadcast was not working but was receiving BG) and trowing error (screenshots above) when I exported the settings from the working follower and imported on a new follower.

The only issue on the not working follower was the BG broadcast, the solution was to export xDrip DB on working follower and import on not working one.
Looks like the Sensor Start was not transferred in the exported settings.

I use xDrip maybe for last 7-9 years, I always find solution for issues I have/had, this issue was new for me, but not one I could not solve :D

This posting is to just make aware of this issue.

**Reply by Navid200** [reply]
My questions are to help me understand if we need to do anything or not.

Do I understand correctly that you use a follower, instead of the master, to upload to AAPS?

**Reply by libxmike** [reply]
no, I use my master for everything, xDrip, AAPS and upload to NS, the follower(s) (in my case) ist/are just for AAPS testing, that's how I notice the issue.
